src/libmongoc/tests/test-mongoc-sample-commands.c

```diff
@@ -197,8 +197,8 @@ test_example_3 (mongoc_database_t *db)
    }
    /* End Example 3 */
    ASSERT_COUNT (4, collection);
-   /* Start Example 3 Post */
 done:
```

Comment:
To confirm: Is removing `done:` from the example intended even though it is referenced in the example? There are `goto done;` statements included in the example above.

Reply:
Discussed over slack. Will leave as-is.